awx/settings/local_settings.py.docker_compose

```diff
@@ -144,9 +141,6 @@ PROXY_IP_WHITELIST = []
 # If set, use -vvv for project updates instead of -v for more output.
 # PROJECT_UPDATE_VVV=True
 
-# Set verbosity for inventory import command when running inventory updates.
-# INVENTORY_UPDATE_VERBOSITY=1
-
```

Comment:
The use of these settings were removed, and setting them here no longer has any effect.